Control Portal/using sAML for single-sign-on to the centurylink platform control portal with windows 2012R2.md

```diff
@@ -0,0 +1,225 @@
+{{{
+  "title": "Using SAML for Single-Sign-On to the CenturyLink Platform Control Portal with Windows 2012R2",
+  "date": "7/21/2015",
+  "author": "Kelly Malloy",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Overview
+The purpose of this guide is to provide step by step instructions for configuring Windows 2012R2 with ADFS 3.0 to authenticate with CenturyLink Cloud SAML.  For simplicity sake, a single server will be used for directory services and certificate services.  Depending on your company's requirements you may choose to use multiple servers for the various roles and services that are required.  If you choose to segregate the roles and services this guide will still provide a useful instruction set for configuration guidelines.
```

Comment:
I looked this up yesterday for some stuff @rathromony was doing but MSFT uses a space `Windows 2012 R2` I assume we should follow what they do